src/StackHCIVM/StackHCIVM.Autorest/examples/Get-AzStackHCIVMAttestationStatuses.md

```diff
@@ -0,0 +1,22 @@
+### Example 1: {{ Add title here }}
```

Comment:
Update the example files and run "build-module.ps1". Don't remove md files under 'docs' folders.